awx/ui/client/src/instance-groups/main.js

```diff
@@ -207,11 +385,11 @@ function InstanceGroupsRun ($stateExtender, strings) {
         resolve: {
             resolvedModels: InstanceGroupsResolve,
             Dataset: ['GetBasePath', 'QuerySet', '$stateParams',
-                function(GetBasePath, qs, $stateParams) {
+                function (GetBasePath, qs, $stateParams) {
```

Comment:
@jakemcdermott @AlexSCorey is this valid js?

Reply:
@shanemcd Yes it seems like t just added a space there.